.travis.yml

```diff
@@ -17,5 +17,6 @@ env:
   matrix:
     - HAWKULAR_VERSION=latest
     - HAWKULAR_VERSION=0.15
```

Comment:
Lets remove this. Newer versions of the driver will not support old-Hawkular releases. If the old versions need patches, we'll fix from the tag.

Reply:
Ok, if that is the case, Can we remove all metrics test instead of skip it?